dags/process_elife_zip_dag.py

```diff
@@ -109,14 +140,20 @@ def wrap_article_in_libero_xml_and_send_to_service(**context):
         '/article/front/article-meta/article-id[@pub-id-type="publisher-id"]'
     )[0].text
 
+    # update image references to .jpg
```

Comment:
looks like a (possibly) temporary change, maybe put it into a separate function so that it's easy to understand what to remove later on?

Reply:
Surely it's easier to remove this code block rather than removing a function call then having to find the function, which is somewhere else, and remove that also?

Reply:
As long as it seems isolated from the rest :+1:

Reply:
(for me it's usually easier to remove a call to a function because it doesn't share the same scope as the calling code, e.g. I don't have to check that variables defined in the function aren't referenced outside of it)

Reply:
ok. I see you point. In this instance, this chunk of code is updating the xml used in the rest of the function so it's isolated.